.github/workflows/rust.yml

```diff
@@ -9,16 +9,10 @@ jobs:
     runs-on: ubuntu-latest
     strategy:
       matrix:
-        rust: ["1.61.0", stable, beta, nightly]
+        rust: [stable, beta, nightly]
```

Comment:
Please keep checks for the MSRV by replacing `1.61.0` -> `1.70.0` rather deleting the lines entirely

Reply:
ah ok